apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -451,7 +452,7 @@ class TopInstructions extends React.Component {
       Math.max(minHeight, maxNeededHeight)
     );
     return maxNeededHeight;
```

Comment:
I don't believe `debounce` works with functions that need to return values; the test failures seem to agree.

Fortunately, it looks like we only rely on this return value in one place:
https://github.com/code-dot-org/code-dot-org/blob/0a26d9cbcfd4056fce0bb37ac24dd0acceb1f177/apps/src/templates/instructions/TopInstructionsCSF.jsx#L262-L266

And we could probably update that to use a callback.